### PR TITLE
Fix styling issues on authentication modules

### PR DESCRIPTION
**Pull request by PabloBuchu**
### Pull Request Description

This PR fixes missing styles on authentication components. It also fixes a bug on pregeneration tailwidn.css. It also encapsulates styles to be applied only under #dashboard div

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [ ] Unit tests have been written where possible.
  - [x] If GUI codebase was changed, the GUI was tested when built using `./run ide build`.

### CODE REVIEW

**Comment by PabloBuchu** [comment]
@wdanilo this PR fixes morning regression by adding missing `-` to output flag (`-o`).

**Comment by PabloBuchu** [comment]
@indiv0 please QA

**Comment by indiv0** [comment]
Styles look fine:
- [x] Under `./run ide build` (no `-authentication`)
- [x] Under `./run ide build` (`-authentication`)
- [x] Under `./run ide watch` (`"authentication": true`)
- [x] Under debug scene (`-startup.entry=text_area`)

QA Approved ✅

**Review by indiv0 — APPROVED** [review]
LGTM

File: app/ide-desktop/lib/content/src/tailwind.css
Comment (wdanilo) [inline app/ide-desktop/lib/content/src/tailwind.css:13]: Why we are copy-pasting code from somewhere. This is a no-go in all of our repos so far, so please, let's find a way not to do it. Question to the person doing code review - why was it not caught?

**Review by wdanilo — CHANGES_REQUESTED** [review]

File: app/ide-desktop/lib/content/src/tailwind.css
Comment (somebody1234) [reply app/ide-desktop/lib/content/src/tailwind.css:13]: i've mentioned elsewhere - but it's difficult to avoid copy-pasting because tailwind does not support prefixes on its preflight. it may be possible using alternative approaches like tailwindcss/nesting and running postcss instead of tailwind

Comment (wdanilo) [reply app/ide-desktop/lib/content/src/tailwind.css:13]: By prefixes, you mean `#dashboard`? Anyway, we need to find a way to do it, copy-pasting generated things is really a no-go. If you want, we can have a call about it and chat about possible solutions.

Comment (somebody1234) [reply app/ide-desktop/lib/content/src/tailwind.css:13]: should be fixed in #6034. can probably be fixed here, *however* Create React App is incompatible with custom postcss processing ([see this link](https://tailwindcss.com/docs/guides/create-react-app)), so this will mean that it won't be possible to make the styles work for `dashboard/` (not that they are currently working)

Comment (somebody1234) [reply app/ide-desktop/lib/content/src/tailwind.css:13]: as a stopgap solution it might be better to just remove tailwind support completely from current develop, since a new implementation should be coming soon anyway

Comment (wdanilo) [reply app/ide-desktop/lib/content/src/tailwind.css:13]: Ok, in order not to block it, I'm OK with any solution. If you want to merge this file for now and we could fix it later, perfect, but lets add a comment about it there, please :)

**Review by wdanilo — APPROVED** [review]

**Comment by wdanilo** [comment]
Please make Q/A before merging! :)

**Comment by somebody1234** [comment]
i think this will be superseded by #6034, so i think the current plan should be to wait for #6034 to be merged and then close this

**Comment by somebody1234** [comment]
hey @PabloBuchu i think this PR is no longer needed?

**Pull request closed without merging**
